grype/match/results.go

```diff
@@ -7,15 +7,15 @@ import (
 	"github.com/scylladb/go-set/strset"
 )
 
-type CPEPackageParameter struct {
+type PackageParameter struct {
```

Comment:
I think a lot of the changes in this file go away if we're not making PURL its own match type or really its own search type, but using an ecosystem search. (Ecosystem search is in `language.go` because most ecosystems are languages. I thought that Bitnami would be more of an outlier in this regard, but it seems not.)